package.json

```diff
@@ -1,20 +1,19 @@
 {
-  "name": "react-hooks-practice-animal-shelter",
+  "name": "rework",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "json-server": "^0.16.3",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-scripts": "^5.0.1",
-    "semantic-ui-css": "1.11",
+    "json-server": "^0.17.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "semantic-ui-css": "^2.5.0",
     "whatwg-fetch": "^3.6.2"
   },
   "scripts": {
     "start": "react-scripts start",
-    "server": "json-server --watch db.json -p 3001",
     "build": "react-scripts build",
-    "test": "echo \".results.json\" && react-scripts test --reporters=@learn-co-curriculum/jest-learn-reporter --reporters=default --watchAll",
```

Comment:
I know I mentioned this in another comment, but just flagging here - we need to keep this `test` script as well as the `server` script.